### PR TITLE
ci(tests): add sweeper to destroy resources after test runs

### PR DESCRIPTION
**Pull request by jeff-held-aiven**
Adds `sweep` target to Makefile, which will run a sweeper to delete resources in a project. This would include:

- services that have "test-" as a prefix for their name
- service integration endpoints
- VPCs

### CODE REVIEW

**Review by Serpentiel — DISMISSED** [review]
Please change the commit's prefix to `ci(tests)`.

File: sweeper/service_integrations.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		}
+	}
+
+	return nil
+}
+
+// isCriticalServiceIntegrationEndpointDeleteError returns true if the given error's status code is not 404
+func isCriticalServiceIntegrationEndpointDeleteError(err error) bool {
+	return err != nil && !avngen.IsNotFound(err)
+}
```
Comment (byashimov) [inline sweeper/service_integrations.go:42]: Can be replaced by `avngen.OmitNotFound`, see below comment.

**Review by byashimov — CHANGES_REQUESTED** [review]

**Review by Serpentiel — APPROVED** [review]
lgtm, @byashimov @jeff-held-aiven I delegate merging to you folks

**Review by byashimov — APPROVED** [review]
lgtm

**Pull request merged**